app-admin/logcheck/logcheck-1.4.2.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd readme.gentoo-r1
+
+DESCRIPTION="Mails anomalies in the system logfiles to the administrator"
+HOMEPAGE="https://logcheck.org/"
+SRC_URI="https://salsa.debian.org/debian/logcheck/-/archive/debian/${PV}/logcheck-debian-${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-debian-${PV}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ~ppc ~sparc x86"
+IUSE="cron"
+# Test (emake system-test) requires access to system logs
+RESTRICT="test"
+
+DEPEND="
+	acct-group/logcheck
+	acct-user/logcheck
+"
+
+RDEPEND="
+	${DEPEND}
+	!app-admin/logsentry
+	app-misc/lockfile-progs
+	dev-lang/perl
+	dev-perl/mime-construct
+	virtual/mailx
+"
+DOC_CONTENTS="
+	Please read the guide at https://wiki.gentoo.org/wiki/Logcheck
+	for installation instructions.
+"
+
+src_prepare() {
+	default
+	# Set version from PV, without using dpkg
+	sed -i -e "s/^VERSION=unknown/VERSION=\"${PV}\"/" "${S}/src/logcheck" || die
+
+	# Add /var/log/messages to checked logs
+	echo "/var/log/messages" >> "${S}/etc/logcheck.logfiles.d/syslog.logfiles" || die
+
+	# QA-fix Remove install of empty dirs to be created at runtime
+	sed -i -e 's*^	install -d $(DESTDIR)/var/lock/logcheck*	#install -d $(DESTDIR)/var/lock/logcheck*' \
+	"${S}/Makefile" || die
+}
+
+src_install() {
+	default
+
+	keepdir /var/lib/logcheck
+
+	dodoc docs/README.*
+	doman docs/logtail.8 docs/logtail2.8
+
+	if use cron; then
+		exeinto /etc/cron.hourly
+		newexe "${FILESDIR}"/${PN}.cron ${PN}
+	fi
+
+	systemd_dounit "${FILESDIR}/${PN}.timer" "${FILESDIR}/${PN}.service"
+
+	readme.gentoo_create_doc
+
+	fowners -R logcheck:logcheck /etc/logcheck /var/lib/logcheck || die
+}
+
+pkg_postinst() {
+	if use cron; then
+		elog
+		elog "Read ${EROOT}/etc/cron.hourly/logcheck.cron to activate hourly check!"
+		elog
+	fi
+
+	if systemd_is_booted; then
+		elog
+		elog "To enable the systemd timer, run the following command:"
+		elog "   systemctl enable --now logcheck.timer"
+		elog
+	fi
+
+	elog "Upstream documentation can be found in /usr/share/doc/${PF}"
```

Comment:
That is usually the case in Gentoo. I don't think that it is common to inform the user about that.